tests/unit/Codeception/Util/StubTest.php

```diff
@@ -2,8 +2,8 @@
 
 declare(strict_types=1);
 
-use \Codeception\Stub;
-use \Codeception\Stub\Expected;
```

Comment:
This file should be deleted altogether.

Codeception\Util\Stub class was renamed to Codeception\Stub and extracted to separate package 3 years ago. I removed class alias Codeception\Util\Stub in 5.0.

The latest version of this test is at https://github.com/Codeception/Stub/blob/master/tests/StubTest.php#L20-L26